src/js/background/identityState.js

```diff
@@ -11,12 +11,23 @@ const identityState = {
       const storeKey = this.getContainerStoreKey(cookieStoreId);
       const storageResponse = await this.area.get([storeKey]);
       if (storageResponse && storeKey in storageResponse) {
+        if (!storageResponse[storeKey].macAddonUUID){
+          storageResponse[storeKey].macAddonUUID = uuidv4();
+          await this.set(cookieStoreId, storageResponse[cookieStoreId]);
```

Comment:
This tries to access `cookieStoreId` in `storageResponse`, but the response contains the `storeKey` at this point.

Reply:
Good catch.